src/wazuh_modules/inventory_harvester/src/systemInventory/elements/hotfixElement.hpp

```diff
@@ -0,0 +1,86 @@
+/*
+ * Wazuh Vulnerability scanner - Scan Orchestrator
```

Comment:
Please fix the name

Reply:
We all overlooked that, is the same in all files. I'll fix some of them